### PR TITLE
New version: ColonyLabs.ScribeDesktopCapture version 4.11.7.0

### PR DESCRIPTION
**Pull request by kris6673**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191514)

### CODE REVIEW

File: manifests/c/ColonyLabs/ScribeDesktopCapture/4.11.7/ColonyLabs.ScribeDesktopCapture.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ColonyLabs.ScribeDesktopCapture
+PackageVersion: 4.11.7
+PackageLocale: en-US
+Publisher: Colony Labs, Inc
+PublisherUrl: https://scribehow.com/
+PublisherSupportUrl: https://support.scribehow.com/hc/en-us/requests/new
```
Comment (stephengillie) [inline manifests/c/ColonyLabs/ScribeDesktopCapture/4.11.7/ColonyLabs.ScribeDesktopCapture.locale.en-US.yaml:9]:
```suggestion
# PublisherSupportUrl: 
```

**Comment by stephengillie** [comment]
Hi @kris6673,

This manifest adds a DisplayVersion to the AppsAndFeaturesEntries that isn't present in previous manifest versions. This entry should be added to the previous versions, or removed from this version.

(Automated response - build 897.)

**Comment by kris6673** [comment]
> Hi @kris6673,
> 
> This manifest adds a DisplayVersion to the AppsAndFeaturesEntries that isn't present in previous manifest versions. This entry should be added to the previous versions, or removed from this version.
> 
> (Automated response - build 897.)

I didnt add a ".0" to the end of the versions like there should have been. sorry for the mistake

File: manifests/c/ColonyLabs/ScribeDesktopCapture/4.11.7.0/ColonyLabs.ScribeDesktopCapture.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+UpgradeBehavior: install
+ProductCode: '{bc907fc3-1a05-44f6-be2c-207b9eda0a1d}'
+ReleaseDate: 2024-11-06
+AppsAndFeaturesEntries:
+- DisplayName: Scribe
+  Publisher: Colony Labs, Inc
+  DisplayVersion: 4.11.7.0
```
Comment (mdanish-kh) [inline manifests/c/ColonyLabs/ScribeDesktopCapture/4.11.7.0/ColonyLabs.ScribeDesktopCapture.installer.yaml:21]:
```suggestion
```

DisplayVersion should not be used when it's equal to PackageVersion. This introduced unnecessary client mapping code

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**